client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -1168,7 +1172,51 @@ export default {
       this.processing = false
     },
 
-    handleRowClicked ({ r: { recordID } }) {
+    handleRowClicked ({ r: { recordID, values, ...record } }) {
+      const { moduleID, namespaceID } = this.recordListModule
+      let prevAndNextRecordFilters = []
+      const field = this.options.presort.split(' ')[0] || 'createdAt'
+      const value = values[field] || record[field]
+
+      const moduleField = this.recordListModule.findField(field)
+
+      prevAndNextRecordFilters.push({
```

Comment:
Instead of creating an array and pushing, just define the array from the start, with the two objects